src/Model/Department/Department.php

```diff
@@ -98,7 +98,7 @@ class Department extends BaseModel
     private $notifyOperators;
 
     /**
-     * @var Operator[]
+     * @var string[]
```

Comment:
What happens if the json encoded string contains a mix of `int` and `string` IDs?

Reply:
It shouldn't break, since this only affects denormalization. an array is just denormalized as is, in whole